internal/nexodus/nexodus.go

```diff
@@ -159,6 +162,7 @@ func NewNexodus(ctx context.Context,
 		version:             version,
 		username:            username,
 		password:            password,
+		skipTlsVerify:       insecureSkipTlsVerify,
```

Comment:
That should be `skipAuthServerTlsVerify` and we should ONLY set that TLS option on the http client used in the go-oidc-agent.